src/govuk/components/checkboxes/template.test.js

```diff
@@ -777,7 +854,25 @@ describe('Checkboxes', () => {
     })
 
     it('passes through fieldset params without breaking', () => {
-      const $ = render('checkboxes', examples['with all fieldset attributes'])
+      const $ = render('checkboxes', {
```

Comment:
"with all fieldset attributes" specified `idPrefix` but didn't test for it as its value in the example was the same as `name` - so you couldn't really tell if it was working or falling back to `name`. 

It would be nice to have a test for `idPrefix` too, what do you think @nickcolley? Or we can do it later.

Reply:
As far as I know idPrefix is only needed for items, which is why the snapshot has not changed, so I think that'd be something to do later.